.github/workflows/performance.yml

```diff
@@ -58,28 +54,75 @@ jobs:
           # BUILD_FOLDERSEARCH=mdn/kitchensink yarn build
           BUILD_FOLDERSEARCH=web/javascript/reference/global_objects/array/foreach yarn build
 
-      - name: Serve and lhci
-        env:
-          LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
+      - name: Serve sample built page
         run: |
           yarn start:static-server &
           sleep 1
           curl --retry-connrefused --retry 5 \
             http://localhost:5000/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach/ > /dev/null
 
-          npm install -g @lhci/cli@next
+      - name: Audit URLs using Lighthouse
+        id: lighthouse_audit
+        uses: treosh/lighthouse-ci-action@v7
+        env:
+          LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
+        with:
+          urls: |
+            http://localhost:5000/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach/
+          # budgetPath: ./budget.json # test performance budgets
+          runs: 1
```

Comment:
Had to think of your recent Lighthouse tweet, shaln't we do multiple runs?

Reply:
If you do multiple runs, I don't think it would be able to calculate a median or something like that. 
Suppose that we run it X times (where X is greater than 1), is it safe to assume that the last one is best? 
There's no CDN caching to take into account. I also don't like that chromium browser has any internal caching that would indicate the n'th run is more realistic. 

What would be interesting is to use the **best** run out of 3. 

The code that generates the comment (as a PR issue comment) uses could perhaps post a little comment for each of the 3 runs and append a human-readable comment that explains that you're supposed to look at the "best one" :)

Another thing we could do is to write our own loop. 
Suppose that `lighthouse` generates 3 .json files, we can manually loop over them and pick out the best run for each category.